bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -184,6 +189,16 @@ spec:
             echo "waiting for db endpoint"
             sleep 5
           done
+
+          hybrid_overlay_flags=
+          if [[ -n "{{.OVNExtensionsEnable}}" ]]; then
```

Comment:
@JacobTanenbaum same as above, lets change the name from extensions to hybrid overlay.

Reply:
changed the name